src/opower/utilities/portlandgeneral.py

```diff
@@ -0,0 +1,81 @@
+"""Portland General Electric (PGE)."""
+
+import re
+from typing import Optional
+import urllib.parse
+
+import aiohttp
+
+from ..const import USER_AGENT
+from ..exceptions import InvalidAuth
+from .base import UtilityBase
+
+class PGN(UtilityBase):
```

Comment:
All the other utilities have a filename and class name that matches the utility website. Use PortlandGeneral here.